### DISCUSSION TITLE
最近在使用Zotero 7时加载不出zotero style

### DISCUSSION THREAD

**Discussion by SlAyOnce** [MuiseDestiny/zotero-style#984] [Q&A]
如上，另外有zotero style的讨论群加吗T。T
<img width="170" alt="截屏2024-05-11 19 21 19" src="https://github.com/MuiseDestiny/zotero-style/assets/169531767/78cda00a-effc-4fea-ae8e-d4c8f5bada5b">

**Comment by SlAyOnce** [comment]
补充：Mac版本的

**Comment by lbc0801** [comment]
windows也不可以
